pages/common/vault.md

```diff
@@ -0,0 +1,27 @@
+# vault
+
+> A CLI to interact with HashiCorp Vault.
+
+- Create a new vault, requiring at least 2 out of 5 keyshares to unseal:
+
+`vault init -key-shares={{5}} -key-threshold={{2}}`
+
+- Unseal the vault by providing one of the keyshares. Repeat with necessary key-shares until unsealed:
```

Comment:
What exactly does "unseal" mean in this context? Would it be possible to add a synonym or equivalent expression in parenthesis?

Also, we probably should use consistent spelling ("keyshares" or "key-shares").

Reply:
When a vault is 'sealed' it means that the master key is missing to access the stored key- and config data. The master key is split up in a number of key shares (by default 5) and can only be reconstructed when a configured number of shares are combined again (by default 3 out of 5).

Unsealing is the process of combining these key shares to construct the original master key again. So when a vault is *sealed*, nobody can access it - not even authenticated clients -  since the master key is missing. When a vault is *unsealed*, it can be used by authenticated clients.

Would `unlocking` as a synonym for `unsealing` be better?